athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/glue/DefaultGlueType.java

```diff
@@ -83,12 +116,12 @@ public static DefaultGlueType fromId(String id)
 
     public static ArrowType toArrowType(String id)
     {
-        DefaultGlueType result = TYPE_MAP.get(id.toLowerCase());
+        ArrowType result = TYPE_MAP.get(id.toLowerCase());
```

Comment:
Why do we need `toArrowType` and `fromId` both seems doing the same thing

Reply:
Yeah, as mentioned on slack, there are callers to both of these methods:

```
~/workspace/aws-athena-query-federation$ git grep "DefaultGlueType\.fromId" .
athena-redis/src/main/java/com/amazonaws/athena/connectors/redis/RedisMetadataHandler.java:        return FieldBuilder.newBuilder(name, DefaultGlueType.fromId(type)).build();
~/workspace/aws-athena-query-federation$ git grep "DefaultGlueType\.toArrowType" .
athena-elasticsearch/src/main/java/com/amazonaws/athena/connectors/elasticsearch/ElasticsearchGlueTypeMapper.java:     * Gets the Arrow type equivalent for the Glue type string representation using the DefaultGlueType.toArrowType
athena-elasticsearch/src/main/java/com/amazonaws/athena/connectors/elasticsearch/ElasticsearchGlueTypeMapper.java:     * conversion routine. The implementation of DefaultGlueType.toArrowType(type) invokes toLowerCase() on the type.
athena-elasticsearch/src/main/java/com/amazonaws/athena/connectors/elasticsearch/ElasticsearchGlueTypeMapper.java:        return DefaultGlueType.toArrowType(type);
athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/glue/GlueFieldLexer.java:    private static final BaseTypeMapper DEFAULT_TYPE_MAPPER = (String type) -> DefaultGlueType.toArrowType(type);
```

Reply:
Just updated the PR with having one of these call the other while retaining both methods (since one of them is used to throw on null and there are callers for this interface).